### PR TITLE
Integration api bot

### PR DESCRIPTION
**Pull request by Jhovannycanaspino**
## Summary
- Completed the integratio api bot

## Screenshot
image

## Trello Card
https://trello.com/c/nAoy2tqF

### CODE REVIEW

File: reactor-training/src/main/java/com/wolox/reactortraining/controller/BotController.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  @ResponseStatus(HttpStatus.CREATED)
+  public void createBook(@RequestBody Boot boot) {
+    RestTemplate restTemplate = new RestTemplate();
+    Boot bootResponse = restTemplate.postForObject(this.urlBoot + "feed", boot, Boot.class);
+  }
+
+  @GetMapping("/talk")
+  public Mono<BootResponse> getBotTalk(@RequestParam(required = false) String name,
+      @RequestParam(required = false, defaultValue = "20") String length) {
+    WebClient webClient = WebClient.create(this.urlBoot);
```
Comment (darkBasic666) [inline reactor-training/src/main/java/com/wolox/reactortraining/controller/BotController.java]: move to property of de class

File: reactor-training/src/main/java/com/wolox/reactortraining/controller/BotController.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    return webClient.get().uri("took?length={length}&name={name}", length, name)
```
Comment (darkBasic666) [inline reactor-training/src/main/java/com/wolox/reactortraining/controller/BotController.java]: Maybe create a services class to manager request and responses, it's a better form to distribute responsibilities

**Review by darkBasic666 — CHANGES_REQUESTED** [review]

**Review by yanzacd — COMMENTED** [review]
There is code that does not make part of this branch

**Review by darkBasic666 — APPROVED** [review]

File: reactor-training/src/main/java/com/wolox/reactortraining/request/Boot.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.wolox.reactortraining.request;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@Getter
+@NoArgsConstructor
+public class Boot {
```
Comment (yanzacd) [inline reactor-training/src/main/java/com/wolox/reactortraining/request/Boot.java:10]: Change name class Boot for Bot, also change name file

File: reactor-training/src/main/java/com/wolox/reactortraining/response/BootResponse.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package com.wolox.reactortraining.response;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class BootResponse {
```
Comment (yanzacd) [inline reactor-training/src/main/java/com/wolox/reactortraining/response/BootResponse.java]: The same before comment

**Review by yanzacd — COMMENTED** [review]
Change each Boot to Bot

**Pull request merged**